src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -362,6 +362,9 @@ function ComingSoonText({ chainId }: { chainId: ChainId }) {
       return <Trans>Coming soon: search and explore tokens on BNB Chain</Trans>
     case ChainId.AVALANCHE:
       return <Trans>Coming soon: search and explore tokens on Avalanche Chain</Trans>
+    case ChainId.BASE:
+    case ChainId.BASE_GOERLI:
```

Comment:
I'm weirdly not seeing this on Base_
<img width="371" alt="Screenshot 2023-07-26 at 3 21 07 PM" src="https://github.com/Uniswap/interface/assets/11512321/56062094-997b-4ace-9a32-fe359687ca20">
Goerli

Reply:
closing the loop on this - it's working for me!